addons/portal/models/mail_message.py

```diff
@@ -105,6 +107,11 @@ def _portal_message_format(self, properties_names, options=None):
                 values['is_message_subtype_note'] = (values.get('subtype_id') or [False, ''])[0] == note_id
             if 'published_date_str' in properties_names:
                 values['published_date_str'] = format_datetime(self.env, values['date']) if values.get('date') else ''
+            values.update({
+                'reactions': message.sudo().reaction_ids._reaction_groups().get(message.id, []),
```

Comment:
This is not going to work in regards to other reaction PR we do, they are only compatible with Store.

If we somehow want to keep `_portal_message_format` in this PR, I'd suggest to just copy/paste old (current) reaction code from before other PRs here.

Or convert `portal_message_format` so that it correctly takes store as param.